lib/lib-table-query/src/main/java/org/sagebionetworks/table/query/model/IntervalLiteral.java

```diff
@@ -28,8 +43,8 @@ public void toSql(StringBuilder builder, ToSqlParameters parameters) {
 	}
 
 	@Override
-	<T extends Element> void addElements(List<T> elements, Class<T> type) {
-		// this is a leaf

```

Comment:
It might be a leaf for a "good" reason, please check the SQLTranslatorUtils and its exception for IntervalLiteral (or the other way around, e.g. maybe we do not need the exception in the SQLTranslatorUtils)

Reply:
The spacial case for SQLTranslatorUtils for IntervalLiteral is still needed.  Since the child of IntervalLiteral is UnsignedInteger and not UnsignedLiteral, including the child in getChildren() does not change anything.